src/translations/components/Section.ts

```diff
@@ -7,6 +7,18 @@ const SectionTexts = {
     placeholder: _('Søk etter adresse eller sted', 'Search for a place'),
     a11yValue: (currentLocation: string) => _(`${currentLocation} er valgt.`),
   },
+  counterInput: {
+    decreaseButton: {
+      a11yLabel: _('Minsk antall'),
+      a11yHint: (text: string, count: number) =>
+        _(`Aktivér for å minske antall ${text} til ${count - 1}`),
```

Comment:
Should we perhaps consider decreasing the count value from where this is called, instead of in the translation text? :thinking:  The reason I'm asking is that we're going to have non-developers working in these files. We could make their job easier by using "preprocessed" values within translations

Reply:
Yeah this can be done. I contemplated both alternatives when implementing and went for the general rule that the component should just send in it's current state, without having knowledge about the text being presented, and it is the texts-file responsibility to create the translation text, and the values for it, based on the current state. When deciding this I did not know that non-developers were going to work on these files. It is totally ok for me that we have as a general rule that all values come preprocessed to the texts-files :+1:

Generally I put quote a bit of logic in the texts-files. Feel free to comment other places where you think the logic would be better if it was outside of the texts-file.

Reply:
I would argue keeping logic out of translation strings as a general rule, to make translation work easier without the need for developer support.

Reply:
I'll rewrite a couple places to keep the logic in the container instead of in the texts file, and then we can compare. It's almost like an additional layer is needed. One layer for keeping the containers separated from text-logic, and one layer for keeping the text-logic separated from the actual texts. Of course this is probably not worth the investment, just a thought :)

Reply:
Might be worth consideration, seems none of the solutions are ideal here.